src/main/java/com/radixdlt/crypto/Signatures.java

```diff
@@ -0,0 +1,66 @@
+package com.radixdlt.crypto;
+
+import java.util.Map;
```

Comment:
Also, not sure this interface is complete yet.  It probably needs at least another method:
```
boolean checkValidSignatures(int countRequired, Hash hash);
```

Reply:
I added:

```java
    boolean hasSignedMessage(Hash message, int requiredMinimumNumberOfValidSignatures);
```

Reply:
Lucky we didn't end up with the last argument being called `requiredMinimumNumberOfValidSignaturesToConfirmThisHashAchievesAnAcceptableQuorum` :)

Reply:
Haha yeah, that could have been a variable name by me :D, still much better than calling all variables `Object a`, `Object b`, `Object i` :D